.php_cs

```diff
@@ -86,38 +87,37 @@ $config->setRules(
         'no_unneeded_final_method' => true,
         'no_unused_imports' => true,
         'no_useless_else' => true,
-        'no_useless_return' => true,
         'no_whitespace_before_comma_in_array' => true,
         'no_whitespace_in_blank_line' => true,
-        'non_printable_character' => ['use_escape_sequences_in_strings' => true],
         'normalize_index_brace' => true,
+        'object_operator_without_whitespace' => true,
         'ordered_imports' => true,
         'phpdoc_add_missing_param_annotation' => true,
         'phpdoc_indent' => true,
         'phpdoc_no_access' => true,
-        'phpdoc_no_empty_return' => true,
         'phpdoc_order' => true,
         'phpdoc_return_self_reference' => true,
         'phpdoc_scalar' => true,
         'phpdoc_separation' => true,
         'phpdoc_single_line_var_spacing' => true,
         'phpdoc_trim' => true,
         'phpdoc_types' => true,
-        'pow_to_exponentiation' => true,
-        'protected_to_private' => true,
-        'psr4' => true,
-        'random_api_migration' => true,
         'return_type_declaration' => true,
-        'self_accessor' => true,
         'semicolon_after_instruction' => true,
         'short_scalar_cast' => true,
-        'simplified_null_return' => true,
```

Comment:
👎

Reply:
My bad, restored in c8aed6faccab3d55a6172742330176c1fc98ebc3.

Reply:
Actually, NOPE.

Explicit null is better, let's leave empty return only to `void`. Also sometimes you simply can't typehint, because function is meant to return different types, like:

```php
function parse(string $value)
{
    switch (mb_strtolower($value)) {
        case 'true':
            return true;

        case 'false':
            return false;

        case 'null':
            return null; // you wouldn't want to remove null here

        default:
            return $value;
    }
}
```